### PR TITLE
Add edit medical information command

### PR DESCRIPTION
**Pull request by sibinhho99-nus**
Usage:
Parameters: `INDEX (must be a positive integer) i/NRIC a/AGE bt/BLOODTYPE md/MEDICATION ht/HEIGHT wt/WEIGHT il/ILLNESSES su/SURGERIES fh/FAMILY_HISTORY ih/IMMUNIZATION_HISTORY gd/GENDER et/ETHNICITY`
Example: `edit 1 i/S1234567L bt/B`

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/medical/EditMedicalCommand.java [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+            + PREFIX_GENDER + "GENDER "
+            + PREFIX_ETHNICITY + "ETHNICITY\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_NRIC + "S1234567L "
+            + PREFIX_BLOODTYPE + "B";
+
+    public static final String MESSAGE_EDIT_MEDICAL_SUCCESS = "Edited Medical Information: %1$s";
+
+    private final Index targetIndex;
+    private final EditMedicalCommand.EditMedicalDescriptor editMedicalDescriptor;
```
Comment (joey-chance) [inline src/main/java/seedu/address/logic/commands/medical/EditMedicalCommand.java:73]: I believe the EditMedicalCommand. is redundant?
```suggestion
    private final EditMedicalDescriptor editMedicalDescriptor;
```

**Review by joey-chance — COMMENTED** [review]
Only one minor fix, else LGTM! 👍

File: src/main/java/seedu/address/logic/commands/medical/EditMedicalCommand.java (hunk 1 above)
Comment (sibinhho99-nus) [reply src/main/java/seedu/address/logic/commands/medical/EditMedicalCommand.java]: Removed

**Review by joey-chance — APPROVED** [review]
LGTM!

**Pull request merged**